src/pages/Sketch/components/Canvas.js

```diff
@@ -31,7 +32,9 @@ function Canvas() {
   const [fontStyle, setFontStyle] = useState("normal");
   const [fontFamily, setFontFamily] = useState("cursive");
 
+  const rc = useRef(null);
```

Comment:
The variable name should be self-explanatory. Update `rc` to something meaningful.

Reply:
rc means rough canvas.

`rc.current` is gonna be use a lot of times in the (I will be implementing drawing handwritten shapes in a lot of places)

I can put a comment saying `//rc represents the sketch-canvas`

what do you say sir?

i can still stick to your suggestion if you want me to do that

Reply:
@ashuvssut It's better if we rename it to `roughCanvas` instead of `rc`. And once this is done, don't forget to remove that comment.